html-aam/index.html

```diff
@@ -1291,6 +1291,83 @@ <h4 id="el-button">`button`</h4>
+        <table aria-labelledby="el-button-select-child">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                <a data-cite="HTML">`button`</a>
+              </td>
+            </tr>
+            <tr>
+              <th>[[wai-aria-1.2]]</th>
+              <td>
+                <div class="general">Not mapped. <a href="#el-button-select-child-comments">See comments</a>.</div>
+                <!-- a button that is a child of a select is not a typical "button" element.
+                     it does not serve as a submit, reset, or generic 'button' type. Rather,
+                     its purpose is directly tied to the rendering of a customized select
+                     serving as the visible indicator/styleable node to serve as the 
+                     invoking element for the select's listbox popup.
+                -->
+              </td>
+            </tr>
+            <tr>
+              <th><a data-cite="core-aam-1.2/#roleMappingComputedRole">Computed Role</a></th>
+              <td class="role-computed">
+                <div class="general">Not mapped. <a href="#el-button-select-child-comments">See comments</a>.</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general"><a href="#el-button-select-child-comments">See comments</a></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general"><a href="#el-button-select-child-comments">See comments</a></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general"><a href="#el-button-select-child-comments">See comments</a></div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general"><a href="#el-button-select-child-comments">See comments</a></div>
+              </td>
+            </tr>
+            <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
+            <tr>
+              <th id="el-button-select-child-comments">Comments</th>
+              <td>
+                <p>
+                  User agents MUST treat the child `button` element of a `select` element as inert. For instance, the UA stylesheet applying <span data-x="css-interactivity">'interactivity: inert'</span> to the button part.
```

Comment:
betting this xref was causing the build fail?

```suggestion
                  User agents MUST treat the child `button` element of a `select` element as inert. For instance, the UA stylesheet applying 'interactivity: inert' to the button part.
```